frontend/app/src/services/websocket/websocket-service.ts

```diff
@@ -45,19 +29,32 @@ class WebsocketService {
       const url = `${protocol}://${websocketUrl}`;
       logger.debug(`preparing to connect to ${url}`);
       this._connection = new WebSocket(url);
-      this._connection.onmessage = event => {
+      this._connection.onmessage = async event => {
         const message: WebsocketMessage<SocketMessageType> = JSON.parse(
           event.data
         );
-        if (message.type === SOCKET_MESSAGE_LEGACY) {
-          const data = message.data;
-          let severity: Severity = Severity.ERROR;
-          let display: boolean = true;
-          if (data.verbosity === MESSAGE_WARNING) {
-            severity = Severity.WARNING;
-            display = false;
-          }
-          notify(data.value, 'Backend', severity, display);
+
+        if (message.type === SocketMessageType.BALANCES_SNAPSHOT_ERROR) {
+          console.log(message);
```

Comment:
lintere caught this, I guess left from debugging

Reply:
Yes, I will take care of it along with adding the legacy checking